book_source/01_introduction/03_how_to_use_book.Rmd

```diff
@@ -44,17 +39,16 @@ We want your ideas, thoughts, comments, and suggestions! This is an open-sourced
 to us in the following ways:
 
 **Github:** [https://github.com/PecanProject/pecan](https://github.com/PecanProject/pecan)
-Serving as the main hub of communication surrounding PEcAn development, check out the issues section to see what is coming down the pipeline. If there is anything you feel you can contribute to, leave a comment.
-​	
+This is the main hub of communication surrounding PEcAn development. Check out the issues section to see known bugs, upcoming features, and ideas for future development. Feel free to comment on existing issues or open new ones with questions, bug reports, feature requests, and/or ideas.
+
 **Slack:** [https://pecanproject.slack.com/](https://pecanproject.slack.com/)
 Slack serves as our day to day mode of communication. To join us in slack you will need to create an account first. This is done in 3 steps:
 
 1. Request an [inivitation](https://publicslack.com/slacks/pecanproject/invites/new) to join slack, this will be send by email to address you provided.
 2. <a id="slackreturn"></a>Check your inbox for an email from Slack with subject "Rob Kooper has invited you to join a Slack workspace". This email should have a link that you can click to join slack.
 3. When you click a webpage will open up that asks you to create an account, once that is done you can login into the slack chatrooms.
 
-
-**Email:** pecanproj[at]gmail.com
+**Email:** pecanproj@gmail.com
```

Comment:
Can we keep it at [at]? Small thing but I'd rather make it hard to get spam emails from scrapers looking for @.

Reply:
I changed it because we had the full email in the Code of Conduct, which I read first. But sure, I'll change it back in both places.